SKA-ScienceDataProcessor/integration-prototype
# Stop checking state when in off

## Description:
The problem is fixed by only checking for database events when the SDP state is not 'off'. 
Fixes issue #92 

## Testing instructions:

1. Set the target_sdp_state to 'off'
2. Remove the docker stack
3. View the master_controller log to verify that it exited cleanly.

## Types of changes

## Checklist:
